<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapmaticWidget</class>
 <widget class="QWidget" name="SnapmaticWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snapmatic Widget</string>
  </property>
  <layout class="QHBoxLayout" name="hlSnapmaticContent">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="SnapmaticFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="hlSnapmatic">
      <item>
       <widget class="QCheckBox" name="cbSelected">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labPicture">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>27</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labPicStr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PHOTO - 00/00/00 00:00:00</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>View picture</string>
        </property>
        <property name="text">
         <string>View</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdCopy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Copy picture</string>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdExport">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Export picture</string>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdDelete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Delete picture</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
